crates/cxx-qt-gen/src/generator/cpp/fragment.rs

```diff
@@ -8,3 +8,8 @@ pub struct CppFragmentPair {
     pub header: String,
     pub source: String,
 }
+
+pub struct CppNamedType {
+    pub ident: String,
+    pub ty: String,
```

Comment:
Should this `ty` be a CxxType?

Reply:
Also, regarding naming, CppNamedType or CxxNamedType?
We should choose one for our internal APIs.
Atm, we're always using Cpp, so maybe change CxxType to CppType?

Reply:
Regarding teh Cpp vs Cxx naming i thought that too. But maybe it's better to do a mass rename at the end as we already have a mixture of cpp and cxx module names etc etc?

Reply:
Changed to be CxxType :-)

Reply:
And renamed things to CppType so we use Cpp for now.